src/emmy/expression/compile.cljc

```diff
@@ -369,6 +373,25 @@
               (gensym-fn 'y)))]
     (rec state)))
 
+(defn- cse
+  "Invokes extract-common-subexpressions on `x`.
```

Comment:
If you write this as [[emmy.expression.cse/extract-common-subexpressions]] it will turn into a hyperlink in our cljdoc. Annoyingly this syntax won't work for vars in clojure.core or anywhere else outside the project, nor will it follow aliases in the namespace... [[var-inside-namespace]] will work without prefix, though.

Reply:
done